src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -36,29 +36,35 @@
 import org.zaproxy.zap.extension.pscan.PassiveScanThread;
 import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
 
+/**
+ * Plugin that scans the content for private IP V4 addresses as well as 
+ * Amazon EC2 private hostnames (for example, ip-12-34-56-78).
+ */
 public class TestInfoPrivateAddressDisclosure extends PluginPassiveScanner {
 
-	/**
-	 * Prefix for internationalised messages used by this rule
-	 */
-	private static final String MESSAGE_PREFIX = "pscanrules.testinfoprivateaddressdisclosure.";
+    /**
+     * Prefix for internationalised messages used by this rule
+     */
+    private static final String MESSAGE_PREFIX = "pscanrules.testinfoprivateaddressdisclosure.";
 	
     private static final String REGULAR_IP_OCTET = "\\b(25[0-5]|2[0-4][0-9]|1?[0-9]{1,2})";
-    private static final String REGULAR_PORTS = "\\b(6553[0-5]|65[0-5][0-2][0-9]|6[0-4][0-9]{4}|[0-5]?[0-9]{0,4})";
+    private static final String REGULAR_PORTS = "\\b(6553[0-5]|65[0-5][0-2][0-9]|6[0-4][0-9]{3}|[0-5]?[0-9]{0,4})";

```

Comment:
hehe... remember my comments in #2834?  (i know i'm an old boring '1stplanner-than-doer' ;)
I'll check that, and also the resulting effect of e.g. `10.0.0.0:65536` -> `10.0.0.0:` in evidence, and commit a next proposal.
BTW, thanks for the regex101-link, didn't know it!